docs/openapi.yaml

```diff
@@ -965,6 +981,26 @@ components:
           type: string
           example: ""
 
+    ScheduledWindow:
```

Comment:
Is it not missing timezone?

Reply:
Good question... I was not sure if we should accept user set the timezone. I think it's not a common use case for Globo, since we use America/Sao_Paulo for almost everything. In other hand, it makes total sense for anyone else using RPaaS aside. I will do it.